changelog.d/8676.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug where an appservice may not be forwarded events for a room it was recently invited to.
```

Comment:
Can you append a `Broke in v1.N` please?

Reply:
I've said v1.22.0 rather than v1.22.0rc1 because I wasn't sure if the distinction mattered for your average changelog reader.

Reply:
I think we usually just put the final release unless we fix it in the same release (broke in v1.22.0rc1 if the fix is in v1.22.0rc2).